src/zzz_od/backend/mcp/app.py

```
@@ -0,0 +1,127 @@
+        """
+        try:
+            image = backend.capture()
+            path = _save_screenshot(image)
+        except Exception as e:  # noqa: BLE001 工具层统一兜底
+            return f"错误: {e}"
+        log.info(f"截图已保存到: {path}")
+        return path
+
+    @mcp.tool()
+    def analyze_screen() -> AnalyzeScreenResult:
+        """分析画面（截图 + OCR），返回结构化结果。
```

Ruff (0.15.20)

[warning] Docstring contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF002)